src/mygrad/nnet/losses/softmax_crossentropy.py

```diff
@@ -26,6 +26,8 @@ def __call__(self, x, y_true):
             Returns
             -------
             The average softmax loss"""
+        if isinstance(y_true, Tensor):
+            y_true = y_true.data
```

Comment:
should we have a function like `np.asarray` that just does this type of check? e.g.:

``` python
def asarray(x):
    if isinstance(x, Tensor):
        x = x.data
    return np.asarray(x)
```

to handle this? We're bound to be doing this check a lot